src/Http/ApiClient.php

```diff
@@ -179,9 +193,12 @@ private function request(
 
     /**
      */
-    private function initialize()
+    private function initialize(string $uri)
     {
-        $this->apiContext->ensureSessionActive();
+        if (!in_array($uri, self::URLS_TO_NOT_ENSURE_ACTIVE_SESSION, true)) {
```

Comment:
Once you have an associative array instead of a list, you can use `isset(self::URLS_TO_NOT_ENSURE_ACTIVE_SESSION[$uri])`

Reply:
👏